docs/docsource/operation/index.rst

```diff
@@ -15,6 +15,7 @@ such as Google Cloud Platform (GCP) or Microsoft Azure.
    :maxdepth: 2
 
    Introduction/index
+   Upgrading_XDM
```

Comment:
Check to be sure this doesn't fit in with Installation. There is a Zenko Upgrade chapter over there. I'm open to moving either to the other location, but we probably shouldn't have one upgrade in Operation and another in Installation.

Reply:
You're right, I moved the file next to Zenko Upgrade.